pkg/publish/openshift.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2021 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package publish
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/ko/pkg/build"
+)
+
+const (
+	// OpenshiftDomain is a sentinel "registry" that represents loading images into
+	// Openshift's internal registry.
+	OpenshiftDomain = "ocp.local"
```

Comment:
It's not really "local" is it? Maybe `ocp.internal`?

Reply:
Depends on how you look at it. It's local in the sense of "local to the cluster". In the same way, `kind.local` may or may not really be local.

I'm not married to it, I just stuck with what was there already. I'm fine with `ocp.internal` as well.